doc/classes/ScrollContainer.xml

```diff
@@ -42,11 +42,20 @@
 	<members>
 		<member name="clip_contents" type="bool" setter="set_clip_contents" getter="is_clipping_contents" overrides="Control" default="true" />
 		<member name="follow_focus" type="bool" setter="set_follow_focus" getter="is_following_focus" default="false">
-			If [code]true[/code], the ScrollContainer will automatically scroll to focused children (including indirect children) to make sure they are fully visible.
+			If [code]true[/code], the ScrollContainer will automatically scroll to focused children (including indirect children) to make sure they are fully visible. Use in combination with [member follow_focus_buffer] to specify how far past the control to scroll.
+		</member>
+		<member name="follow_focus_buffer" type="Vector2" setter="set_follow_focus_buffer" getter="get_follow_focus_buffer" default="Vector2(0, 0)">
+			When [member follow_focus] is [code]true[/code], specifies how far past the focused control to scroll, so that the focused control is not barely visibile.
```

Comment:
```suggestion
			When [member follow_focus] is [code]true[/code], specifies how far past the focused control to scroll, so that the focused control is not barely visible.
```

Aside from the typo, "is not barely visible" sounds a bit awkward to me.

Reply:
Original behaviour is to just barely scroll the scroll container enough to have the focused element in view, this allows you to set a buffer to scroll _past_ the focused container. Not sure what a better wording is.

Maybe even just "When [member follow_focus] is [code]true[/code], specifies how far past the focused control to scroll."